src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -124,7 +144,13 @@ const ChartsHolder = React.memo(({ region, chartId }: ChartsHolderProps) => {
 
     scrollToChart();
     scrollToRecommendations();
-  }, [chartId, metricRefs, isRecommendationsShareUrl, scrolledWithRef]);
+  }, [
+    chartId,
+    metricRefs,
+    isRecommendationsShareUrl,
+    scrolledWithRef,
+    chartBlockRefs,
```

Comment:
Seems like this isn't used in the effect?  Is it actually necessary?

Reply:
oh yeah i think this is a remnant of me trying to get the share links working